packages/pure/src/endpoints/backup.types.ts

```diff
@@ -22,7 +22,10 @@ export interface StartBackupResponseBody {
   id: string
 }
 
-export type GetBackupDeviceStatusRequestConfigBody = StartBackupResponseBody
+export interface GetBackupDeviceStatusRequestConfigBody {
+  id: string
+  category?: BackupCategory
+}
```

Comment:
I'd like to ask just to be sure. Does this PR not provide any breaking change for the near Mudita Center release? 👉 1.6.1? Do these changes require a new OS?

Reply:
@dkarski This functionality protected via https://github.com/mudita/mudita-center/pull/1011/files#diff-206c1580b0150324ef6656592757f01f3931b93d00ad0ad3118b156a27cf160cR46